packages/node_modules/@ciscospark/redux-module-conversation/src/actions.js

```diff
@@ -330,7 +330,7 @@ export function loadPreviousMessages(conversationId, maxDate, spark) {
       conversationId,
       lastActivityFirst: true,
       limit: 20,
-      maxDate
+      maxDate: Date.parse(maxDate)
```

Comment:
May want to add an input check here to make sure we get a real date back, and not call the API if we don't.